kodi_addon_checker/check_addon.py

```diff
@@ -79,6 +83,9 @@ def start(addon_path, repo_addons, config=None):
 
             _check_for_invalid_xml_files(addon_report, file_index)
 
+            if config.is_enabled("check_for_existing_addon"):
```

Comment:
I think we should be able to drop this check. As we would try to get that config file down to zero. Having both a config file and command line parameters might be very confusing in the long run.